tutor/test_t1_58_ManageEcosystems.py

```diff
@@ -67,6 +82,7 @@ def tearDown(self):
             )
         try:
             self.admin.delete()
+            self.content.delete()
```

Comment:
Using `existing_driver` means that `delete()` only needs to be called on `admin`